scss/_alert.scss

```diff
@@ -54,18 +54,13 @@
 
 
 // scss-docs-start alert-modifiers
-// Generate contextual modifier classes for colorizing the alert.
-
+// Generate contextual modifier classes for colorizing the alert
 @each $state, $value in $theme-colors {
```

Comment:
`$value` becomes unused here

Reply:
These have to stay I think? I get this error in the Terminal when I remove it from the `@each` loop:

<img width="608" alt="CleanShot 2022-11-19 at 13 41 09@2x" src="https://user-images.githubusercontent.com/98681/202872607-09b61b31-5806-4277-862e-3b82131ad64f.png">

Reply:
We can do probably do something like `@each $state in map-keys($theme-colors) {` but that's a detail. Nevermind :)